rosdep/python.yaml

```diff
@@ -2493,6 +2493,28 @@ python-levenshtein:
   fedora: [python-Levenshtein]
   gentoo: [dev-python/python-levenshtein]
   ubuntu: [python-levenshtein]
+python3-pylibdmtx:
```

Comment:
```
ERR: list out of alphabetical order line 2518.  'python-libpcap' should come before 'python3-pylibdmtx'
```

This entire python3 key should be moved in the file alongside other python3 keys.

Reply:
sorry for taking so long to fix this, slipped my mind.
I have moved the section to its correct place in alphabetical order